test/HotKeys/focus-only/ChangingKeyMapAndHandlersAfterMount.spec.js

```diff
@@ -40,10 +40,14 @@ describe('Changing keyMap and handlers after mount:', function () {
 
       it('then the new sequence is ignored', function() {
         this.targetElement.keyDown(KeyCode.B);
+        this.targetElement.keyPress(KeyCode.B);
```

Comment:
🐥 These tests didn't bother to include the `keyup` event as it wasn't significant. Now that submatches are disabled by default, and key combinations cannot overlap, we need to add the `keyup` events, to close off the first key combination and start a new one.